kpi/deployment_backends/kobocat_backend.py

```diff
@@ -722,7 +803,30 @@ def __kobocat_proxy_request(kc_request, user=None):
         return session.send(kc_request.prepare())
 
     @staticmethod
-    def __prepare_as_drf_response_signature(requests_response):
+    def __prepare_duplication_response(kc_response, uuid_):
+        """
+        Prepares response from OpenRosa to be fed to
+        `__prepare_as_drf_response_signature` for correct signature formatting
+        for drf.
+        """
+
+        # parsing xml response from OpenRosa to get response message
+        parsed_response = ET.fromstring(kc_response._content)
+        detail_obj = parsed_response.find(
+            '{http://openrosa.org/http/response}message'
+        )
+        detail_ = _(detail_obj.text) if hasattr(detail_obj, 'text') else \
+                _('Something went wrong with duplicating the submission.')
```

Comment:
At this point, since there's nothing that the user can do, and we moreover cannot provide any useful information about the problem, we should fail with a 500 error. Example: https://github.com/kobotoolbox/kpi/blob/f52f77d9f6350cc5ffa8d89414daa13a1a536da2/kpi/deployment_backends/kobocat_backend.py#L155-L160

A new exception type would be needed, but it can be an empty class (`pass` only) that inherits from `Exception`. This has the added benefit of logging the error in Sentry where we can inspect the problem. (To be fair, you can also use `logging` to write to Sentry without raising an exception)